xc7/make/arch_define.cmake

```diff
@@ -83,6 +83,7 @@ function(ADD_XC7_ARCH_DEFINE)
         --pcf \${INPUT_IO_FILE} \
         --eblif \${OUT_EBLIF} \
         --top \${TOP} \
+        --allow_orphan_sinks \
```

Comment:
Can we try dropping this option?  I don't believe it is required anymore for fasm2bels to work.

Reply:
Done, should I locally check that fasm2bels works locally or is it already tested in CI?

Reply:
Locally.  We cannot test ROI-less stuff until the lookahead search is updated.